app/views/works/_hidden_fields.html.erb

```diff
@@ -33,6 +33,10 @@
   <%= c.hidden_field :title, :value => "#{@chapter.title}" %>
   <%= c.hidden_field :published_at, :value => "#{@chapter.published_at}" %>
 <% end %>
+<%= fields_for "work[series_attributes]", @serial do |s| %>
```

Comment:
When the form errors out, can we retrieve the unsaved series info from `work[series_attributes]`, instead of adding new hidden fields? Similar to:

https://github.com/otwcode/otwarchive/blob/bedbbef8c43585d97b5a56062355921e4c8f5fa7/app/helpers/works_helper.rb#L58-L63

Reply:
I've changed the code to get series id/title from work[series_attributes], but hidden fields are still necessary to save series when we save changes from Preview page, otherwise series_attributes data is not sent to the server

Reply:
Makes sense, thanks!